src/components/Expander/Expander.jsx

```diff
@@ -15,9 +15,34 @@ const {
 	func,
 	node,
 	object,
+	oneOf,
 	string,
 } = React.PropTypes;
 
+export const ChevronButton = createClass({
```

Comment:
I'm not sure this Component is necessary, it seems to be just a wrapper for ChevronIcon.